<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="client_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="server_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Servidor</string>
    </property>
   </widget>
   <widget class="QLabel" name="host_label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>390</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Porta:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="host_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_text">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
